packages/apps/src/SideBar/index.tsx

```diff
@@ -17,13 +17,10 @@ import translate from '../translate';
 import Item from './Item';
 
 type Props = I18nProps & {
-  children?: React.ReactNode
+  children?: React.ReactNode;
 };
 
-const LOGOS: Map<string | undefined, any> = new Map([
-  ['polkadot', polkadotLogo],
-  ['substrate', substrateLogo]
-]);
+const LOGOS: Map<string | undefined, any> = new Map([['polkadot', polkadotLogo], ['substrate', substrateLogo]]);
```

Comment:
Prefer the previous version, where the Array entries are each on a single line. (The all-in-one gets very unwieldy when there are multiples)

Reply:
Okay, I will see to the changes. I have some auto plugins installed in my IDE and that is why everything seems dismantled